CRM/Contribute/Form/AdditionalPayment.php

```diff
@@ -185,7 +185,7 @@ public function setDefaultValues() {
     }
 
     if ($this->_refund) {
-      $defaults['total_amount'] = abs($this->_refund);
+      $defaults['total_amount'] = number_format(abs($this->_refund), 2);
     }
```

Comment:
Certain currencies support more than 2 decimal places so not sure we should be hardcoding 2 here.  See https://github.com/civicrm/civicrm-core/pull/10641/files#diff-9dacd15500c58030b03a70b7501fa5e5R87 or CRM_Utils_Money::format with onlyNumber=TRUE might be more appropriate?